### PR TITLE
release 2.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
We should probably wait for this until https://github.com/voxpupuli/puppet-nodejs/pull/248 is merged?

**Comment by bastelfreak** [comment]
https://github.com/voxpupuli/puppet-nodejs/pull/251 contains #248 and got merged.

File: metadata.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
       "operatingsystemrelease": [
         "10.04",
         "12.04",
-        "14.04"
+        "14.04",
+        "16.04"
       ]
```
Comment (igalic) [inline metadata.json]: no mention of Fedora here?

Comment (bastelfreak) [inline metadata.json]: Got lost in the PR for fedora, I added it..

**Pull request merged**